client/landing/stepper/hooks/use-save-query-params.ts

```diff
@@ -0,0 +1,31 @@
+import { useDispatch } from '@wordpress/data';
```

Comment:
`Non-blocking`:

I'm actually surprised something like this didn't exist prior to our work, but a cursory search in the codebase didn't turn anything up for me.

The closest thing I found was `client/lib/analytics/utils/save-coupon-query-argument.js`, which saves a coupon query arg to local storage.

Thanks for generalizing and implementing this @niranjan-uma-shankar 🙂